ovgosd.c

```diff
@@ -886,8 +886,12 @@ class cOvgCmdDestroySurface : public cOvgCmd
 			vgDestroyImage(m_target->image);
 		}
 		delete m_target;
-		return true;
+		m_targetRef = NULL;
+		return ok;
 	}
+
+private:
+	cOvgRenderTarget *&m_targetRef;
```

Comment:
what is `m_targetRef` used for?

Reply:
The assignment `m_targetRef = NULL` in `cOvgCmdDestroySurface::Execute()` will be observed by `cOvgThread::DestroySurface()`:
```c++
	void DestroySurface(cOvgRenderTarget *&buffer)
	{
		DoCmd(new cOvgCmdDestroySurface(buffer));
		m_commandsMutex.Lock();
		while (buffer)
			m_commandsLoop.Wait(m_commandsMutex);
		m_commandsMutex.Unlock();
	}
```

Reply:
ah, got it!